src/OrleansServiceFabricUtils/FabricMembershipOracle.cs

```diff
@@ -50,9 +51,9 @@ internal class FabricMembershipOracle : IMembershipOracle, IFabricServiceStatusL
             ILocalSiloDetails localSiloDetails,
             GlobalConfiguration globalConfig,
             IFabricServiceSiloResolver fabricServiceSiloResolver,
-            Factory<string, Logger> loggerFactory)
+            ILoggerFactory loggerFactory)
         {
-            this.log = loggerFactory("MembershipOracle");
+            this.log = new LoggerWrapper<FabricMembershipOracle>(loggerFactory);
```

Comment:
nit: just inject the generic version

Reply:
will do . I was cleaning those "should inject the generic version" stuff in previous commit. I guess I missed MemebershipTable, ReminderTable and GatewayList{Provider, which was implicit created from DI. Wil address these